packages/devtools_app/lib/src/memory/memory_tracker_model.dart

```diff
@@ -248,7 +248,7 @@ class TreeTracker {
                 ],
               ),
               separatorBuilder: (context, index) =>
-                  Divider(color: devtoolsGrey[400], height: 8),
```

Comment:
nit: make 12 a constant in the list of devtools spacing constants. Could be defaultSpacerHeight or similar.

Reply:
Done.